src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.field {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  flex-basis: 33.33%;
+  flex-grow: 1;
+  height: 300px;
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.field__block_1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.field__block_2 {
+  background-color: rgb(205, 0, 0);
+}
+
+.field__block_3 {
+  background-color: rgb(154, 0, 0);
+}
+
+.field__block_4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.field__block_5 {
+  background-color: rgb(48, 0, 0);
+}
+
+.field__block_6 {
+  background-color: rgb(0, 0, 0);
+}
+@media (min-width: 300px) {
+  .field {
+    flex-basis: 100%;
+  }
```

Comment:
these styles do nothing. delete it. makes sense to add flex-basis only in situation when you hame more than one block. Here you have only one field and its children